src/pages/index.tsx

```diff
@@ -18,11 +21,44 @@ const IndexPage: FunctionComponent = () => {
       />
       <main className="bg-gray-50 dark:bg-gray-900 sm:-my-5 -my-3 -mx-5 p-5">
         <div className="max-w-screen-xl mx-auto">
-          <Home />
+          <Home sections={sections} />
         </div>
       </main>
     </>
   )
 }
 
 export default IndexPage
+
+const digitalGardeningQuery = groq`
+*[_type == 'resource' && slug.current == "digital-gardening-for-developers"][0]{
+  title,
+  description,
+  'illustration': images[label == 'eggo'][0]{
+    url,
+    alt
+  },
+  'quote': content[title == 'quote'][0]{
+    description
+  },
+  resources[]{
+    title,
+    byline,
+    'name': content[title == 'name'][0].description,
+    'path': resources[]->[0].path,
+    'image': resources[]->[0].image
+  }
+}
```

Comment:
thanks to @theianjones we don't need to transform the data on the front end.

This queries the feature data for this section. When we have more section (paths) we want to build out, we can parameterize this query.

Reply:
suggestion: in my heart of hearts I think we need to consider features as pages